extensions/dashlane-vault/src/lib/authcli.ts

```diff
@@ -0,0 +1,27 @@
+import { environment } from "@raycast/api";
+import { execa } from "execa";
+import { chmod } from "fs/promises";
+import { join } from "path";
+
+const executeCommand = async (args: string[]) => {
+  const command = join(environment.assetsPath, "auth");
```

Comment:
Could you move this constant outside of the function similar to `CLI_PATH` in `dcli.ts`. This path will never change during runtime.